src/playwright/github/jobs/playwright-job.ts

```diff
@@ -0,0 +1,19 @@
+import {Job} from 'projen/lib/github/workflows-model'
```

Comment:
Apart from the files copied from the `common` folder, this is the only one remaining, so we can safely remove the `jobs` folder and store this file one level upper.